lizmap/modules/lizmap/classes/qgisServer.class.php

```diff
@@ -30,6 +30,13 @@ public function __construct()
 
     public function getPlugins($project)
     {
+        $key = 'qgis/server/plugins';
+        $key = jCache::normalizeKey($key);
+        $plugins = jCache::get($key);
+        if ($plugins !== false) {
```

Comment:
The implementation of the plugin "file" for jCache is not consistent with other plugins (my fault). It may return null instead of false if the cache does not exist. Prefer to check both `false` and `null`.

Reply:
Checked that `$plugins` is not `null` and not `false`.